docs/sql-guide/functions/function-totimestamp.md

```diff
@@ -0,0 +1,78 @@
+---
+title: TOTIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# TOTIMESTAMP() function
+
+The `TOTIMESTAMP()` function converts the given integer value to a timestamp and returns it. It uses the [unix epoch](https://www.unixtutorial.org/unix-epoch/) as the base and calculates date-time equivalent of the integer.
+
+## Syntax
+
+```
+TOTIMESTAMP(int_exp, timeunit)
+```
+
+## Arguments
+
+
+| Argument | Description | Data type |
+|---|---|---|
+| `int_exp` | Required `int` value or expression to be converted to a `timestamp`. | `int` |
+| `timeunit` | Optional `string` value specifying the granularity of the input `int_exp`, defaults to `s`. see [TIMEUNIT values](#timeunit-value).| `string` |
```

Comment:
| `timeunit` | string | String value that specifies the unit of time to convert.  |  Yes | [Time units](ADD HEADING BELOW) |